### PR TITLE
docs: create contributing guide

### PR DESCRIPTION
**Pull request by jzapiorkowski**
## Description

create contributing guide

it should contain info about how to name branches
types of branches:
- feat
- test
- fix
- docs
- style
- refactor
- build
- chore
- perf

add info about how to merge branches
add info about how to name pull requests

## Clickup Ticket

https://app.clickup.com/t/33vaz6n

### CODE REVIEW

File: docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+There are following types of branches:
+
+- `feat` for branches that introduce a new feature to the codebase
+- `fix` for branches that patch a bug in the codebase
+- `test` for branches that add missing tests or correcting existing tests
+- `docs` for branches that affect documentation only
+- `refactor` for branches with code change that neither fixes a bug nor adds a feature
+- `style` for branches that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
+- `build` for branches that affect build components like build tool, ci pipeline, dependencies
```
Comment (wgola) [inline docs/CONTRIBUTING.md:18]: I think 'ci' should be in capital letters

File: docs/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+- `chore` - for branches with miscellaneous commits e.g. modifying .gitignore
+- `perf` - for branches with special `refactor` commits, that improve performance
+
+So new feature branch will have a name e.g. `feat/create-footbar` \
+You can merge your branches after getting at least one review, but remember that every new feature should be fully tested
+
+## Before merging your branch to master check also release guide
```
Comment (wgola) [inline docs/CONTRIBUTING.md:25]: I think 'please also check' sounds better than 'check also'

File: docs/CONTRIBUTING.md (hunk 1 above)
Comment (jzapiorkowski) [reply docs/CONTRIBUTING.md]: resolved

File: docs/CONTRIBUTING.md (hunk 2 above)
Comment (jzapiorkowski) [reply docs/CONTRIBUTING.md]: resolved

**Review by wgola — APPROVED** [review]
LGTM

**Pull request merged**